examples/multimodal/multimodal_llm/neva/conf/neva_config.yaml

```diff
@@ -186,12 +187,22 @@ model:
     num_workers: 8
     dataloader_type: cyclic
     data_path:
+      # could be a path to a single file or a list of files for data blending like below
+      # - /path/to/json
+      # - /path/to/json
+    global_batch_size: ${model.global_batch_size}
+    micro_batch_size: ${model.micro_batch_size}
+    concat_sampling_probabilities: null
```

Comment:
will it work if user define two json in path but set this to null? If not, add a assertion later

Reply:
adjusted codes in `neva_model` session